src/mhd2xds.coffee

```diff
@@ -1,8 +1,25 @@
 js2xml = require 'js2xmlparser'
-uuid = require 'node-uuid'
+uuid = require 'uuid'
+dom = require('xmldom').DOMParser
+xpath = require 'xpath'
 xds = require './xds'
 
-exports.mhd1Metadata2Xds = (metadata) ->
+exports.mhd1Metadata2XdsForMomconnect = (metadata, cda) ->
+  cda = cda.trim()
+  doc = new dom().parseFromString(cda)
+
+  select = xpath.useNamespaces
+    'v3': 'urn:hl7-org:v3'
+
+  birthDate = select 'string(//v3:ClinicalDocument/v3:recordTarget/v3:patientRole/v3:patient/v3:birthTime/@value)', doc
+  birthDate = birthDate.replace '%', ''
+
+  authorId = select 'string(//v3:ClinicalDocument/v3:author/v3:assignedAuthor/v3:id/@extension)', doc
+  if authorId.length is 0 then authorId = 'unknown'
+  authorAssigningAuthorityId = select 'string(//v3:ClinicalDocument/v3:author/v3:assignedAuthor/v3:id/@root)', doc
```

Comment:
Would it be possible to add handling for multiple authors? Specifically for this there's a second author capturing the device information (software source):
```
<author>
  <time value="201501011300"/>
  <assignedAuthor>
    <id root="9a560d61-85f1-4d7b-8ee2-090d2900f836"/>
    <assignedAuthoringDevice>
      <code code="applicationCode" codeSystem="56877fb7-e3a9-4ad5-bfb5-64d48a045e83"/>
      <softwareName>software name</softwareName>
    </assignedAuthoringDevice>
  </assignedAuthor>
</author>
```